src/ray/common/event_stats.cc

```diff
@@ -211,11 +231,15 @@ std::string EventTracker::StatsString() const {
     if (entry.second.running_count > 0) {
       event_stats_stream << ", " << entry.second.running_count << " running";
     }
-    event_stats_stream << "), CPU time: mean = "
+    event_stats_stream << "), Execution time: mean = "
                        << to_human_readable(entry.second.cum_execution_time /
                                             static_cast<double>(entry.second.cum_count))
                        << ", total = "
-                       << to_human_readable(entry.second.cum_execution_time);
+                       << to_human_readable(entry.second.cum_execution_time)
+                       << ", Queueing time: mean = "
+                       << to_human_readable(entry.second.cum_queue_time /
```

Comment:
Q: should we also track the max time here? I feel like max time is actually pretty useful. 

Maybe doesn't have to be included in this PR

Reply:
Good point. Let me add in this PR.